src/renderer/features/settings/components/playback/lyric-settings.tsx

```diff
@@ -116,6 +124,47 @@ export const LyricSettings = () => {
             isHidden: !isElectron(),
             title: t('setting.lyricOffset', { postProcess: 'sentenceCase' }),
         },
+        {
+            control: (
+                <Select
+                    data={languages}
+                    value={settings.targetLanguage}
+                    onChange={(value) => {
+                        setSettings({ lyrics: { ...settings, targetLanguage: value } });
+                    }}
+                />
+            ),
+            description: t('Target language for translation'),
+            isHidden: !isElectron(),
+            title: t('setting.targetLanguage', { postProcess: 'sentenceCase' }),
+        },
+        {
+            control: (
+                <Select
+                    data={['Microsoft Azure', 'Google Cloud']}
+                    value={settings.apiProvider}
+                    onChange={(value) => {
+                        setSettings({ lyrics: { ...settings, apiProvider: value } });
+                    }}
+                />
+            ),
+            description: t('Api provider for translation'),
+            isHidden: !isElectron(),
+            title: t('setting.apiProvider', { postProcess: 'sentenceCase' }),
+        },
+        {
+            control: (
+                <TextInput
+                    value={settings.apiKey}
+                    onChange={(e) => {
+                        setSettings({ lyrics: { ...settings, apiKey: e.currentTarget.value } });
+                    }}
+                />
+            ),
+            description: t('Api key for translation (Support global service endpoint only)'),
```

Comment:
Modified

Reply:
This should be using a translation key instead of the raw string.

Reply:
> This should be using a translation key instead of the raw string.

Oh sorry, forgot that, commited.